docs/design/properties_code_point_trie.md

```diff
@@ -22,6 +22,12 @@ String Properties return a code point or sequence of code points. Numeric proper
 
  For returning useful properties to the user, we should exclude contributory and provisional properties from APIs.
 
+### Notes About Properties
+
+As [it is defined](https://www.unicode.org/reports/tr44/#General_Category_Values), the General_Category property of a code point "provides for the most general classification of that code point. It is usually determined based on the primary characteristic of the assigned character for that code point. For example, is the character a letter, a mark, a number, punctuation, or a symbol, and if so, of what type?" This useful high-level information that applies in multiple use cases, so the values are stored via bit mask in ICU4C. In the same section 5.7.1 of UAX #44, a table lists the values, including aliases representing the union of other values (LC, L, M, N, P, S, Z, C). For example, the long name of the value N is Number, which is the union of Nd (Decimal_Number), Nl (Letter_Number), and No (Other_Number). These aliases are provided as conveniences for use cases such as regex. However, "these aliases for groupings of General_Category values do not occur in UnicodeData.txt, which instead always specifies the enumerated subtype for the General_Category of a character." As such, the aliases do not appear in the ICU4C enum [`UCharCategory`](https://unicode-org.github.io/icu-docs/apidoc/dev/icu4c/uchar_8h.html#a6a2dbc531efce8d77fdb4c314e7fc25e) for General_Category values.
```

Comment:
```suggestion
As [it is defined](https://www.unicode.org/reports/tr44/#General_Category_Values), the General_Category property of a code point "provides for the most general classification of that code point. It is usually determined based on the primary characteristic of the assigned character for that code point. For example, is the character a letter, a mark, a number, punctuation, or a symbol, and if so, of what type?" In the same section 5.7.1 of UAX #44, a table lists the values, including aliases (LC, L, M, N, P, S, Z, C) representing the union of multiple other values. For example, the long name of the value N is Number, which is the union of Nd (Decimal_Number), Nl (Letter_Number), and No (Other_Number). These aliases are provided as separate multi-value constants. Any one code point has one GeneralCategory. The multi-value constants are handy for "is the character a letter or number" etc. testing. TODO: Decide bit set integers vs. something like EnumSet, see PR comment.
```
In particular, "This useful high-level information that applies in multiple use cases, so the values are stored via bit mask in ICU4C." is not true. ICU stores the UCharCategory, just like UnicodeData.txt shows, and returns that as a bit flag when desired, basically via `(1 << gc)`.